android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt

```diff
@@ -7,6 +7,7 @@ enum class OneWelcomeWrapperErrors(val code: Int, val message: String) {
     AUTHENTICATOR_NOT_FOUND(8004, "The requested authenticator is not found"),
     HTTP_REQUEST_ERROR(8011, "OneWelcome: HTTP Request failed internally."),
     ERROR_CODE_HTTP_REQUEST(8013, "OneWelcome: HTTP Request returned an error code. Check Response for more info."),
+    UNAUTHENTICATED_IMPLICITLY(8035, "The requested action requires you to be authenticated implicitly"),
```

Comment:
Please use a more descriptive error name. `USER_NOT_AUTHENTICATED_IMPLICITLY` ? might be a bit long, but I'm sure this is better or you could find a smaller one which is also descriptive.

Reply:
done